Sources/DownloadService.swift

```diff
@@ -22,7 +22,7 @@ private let session: NSURLSession = {
   return NSURLSession(configuration: config)
 }()
 
-class DownloadService {
+internal class DownloadService {
```

Comment:
FYI, internal is the default permission. So no need to mark them when using .frameworks